### ISSUE TITLE
Some errors using Google Java Style configuration

### ISSUE THREAD

**Issue by davidhyk** [checkstyle/checkstyle#599]
Hi, there.

I'm using Checkstyle-IDEA plugin on Android Studio. The plugin uses Checkstyle 6.1.1. 

I imported Checkstyle's Google Java Style configuration of master branch, but I've got following errors: 

```
org.infernus.idea.checkstyle.exception.CheckStylePluginException: <html><b>The CheckStyle rules file could not be loaded.</b><br>Property 'fileExtensions' in module Checker does not exist, please check the documentation</html>
	at org.infernus.idea.checkstyle.checker.CheckerFactory.blacklistAndShowMessage(CheckerFactory.java:234)
	at org.infernus.idea.checkstyle.checker.CheckerFactory.createChecker(CheckerFactory.java:188)
	at org.infernus.idea.checkstyle.checker.CheckerFactory.getOrCreateCachedChecker(CheckerFactory.java:98)
	at org.infernus.idea.checkstyle.checker.CheckerFactory.getChecker(CheckerFactory.java:73)
	at org.infernus.idea.checkstyle.checker.CheckerFactory.getChecker(CheckerFactory.java:41)
	at org.infernus.idea.checkstyle.ui.LocationDialogue.testLoadOfFile(LocationDialogue.java:197)
	at org.infernus.idea.checkstyle.ui.LocationDialogue.access$800(LocationDialogue.java:24)
	at org.infernus.idea.checkstyle.ui.LocationDialogue$NextAction.actionPerformed(LocationDialogue.java:242)
```

It seems like the configuration file's problem(src/main/resources/google_checks.xml).

Please fix this problem.

Regards,
David

**Comment by romani** [comment]
That feature will be released in 6.3, please use checkstyle config from 6.1.1 release. https://github.com/checkstyle/checkstyle/releases

https://github.com/checkstyle/checkstyle/blob/18f6ebbcad23e88e3ae30fc79be464b8b7772a0d/google_checks.xml


Sorry it become embeded in final jar only in 6.2 releaese.

**Comment by davidhyk** [comment]
It works! Thanks for reply :)
